src/ripple/json/json_value.h

```diff
@@ -227,9 +227,9 @@ class Value
      * Json::Value aValue( StaticString("some text") );
      * \endcode
      */
-    Value ( const StaticString& value );
-    Value ( std::string const& value );
-    Value ( bool value );
+    explicit Value ( const StaticString& value );
+    explicit Value ( std::string const& value );
+    explicit Value ( bool value );
```

Comment:
When building an `rpc` overloaded handler for WSClient I got a bunch of ambiguities of string-like objects getting implicitly converted to `Json::Value` instead of into `std::string`, causing run-time errors.  That pretty much scared me to death since the meaning of a  `Json::Value`  containing a string value means something quite different than a `std::string`.